app/src/main/java/com/sdp/movemeet/Backend/Serialization/ActivitySerializer.java

```diff
@@ -17,7 +17,7 @@
     public static final String TITLE_KEY = "title";
 
     public static final String NPART_KEY = "numberParticipant";
-    public static final String PARTICIPANTS_KEY = "participantsId";
+    public static final String PARTICIPANTS_KEY = "participantId";
```

Comment:
the name of the attribute is in plurial but the value singular ?

Reply:
Yes, the fact is that the Firebase Firestore field has initially been set in the singular ("participantId") and despite my efforts, I unfortunately could not find a way to modify it. But you're right, that's very confusing and this reference should be in the plural in Firebase Firestore... Since Kepler initially created the different fields of the sport activities on Firebase, we might have to ask him if we have a way to modify the name of some fields!